### DISCUSSION TITLE
Transformations: Variables should work in 'Filter data by values' condition

### DISCUSSION THREAD

**Discussion by imReker** [grafana/grafana#39636] [Dashboards, Panels & Visualizations]
**What happened**:
Variables not work in 'Filter data by values' condition.

**How to reproduce it (as minimally and precisely as possible)**:
1. Add a 'Filter data by values' transform.
2. Input variable in condition value.

**Environment**:
- Grafana version:Grafana v8.1.2 (103f8fa094)

**Comment by ivanahuckova** [comment]
@imReker this seems like a feature request. Would you mind creating new discussion item https://github.com/grafana/grafana/discussions/new. Or if I am wrong and this is bug, could you please provide more information.

**Comment by imReker** [comment]
I'm not sure if variables designed to work in transformations, my personal opinion that variables 'should' work everywhere in panel. Maybe this is a feature request.

**Comment by ChrisStrykesAgain** [comment]
My bad for not searching, but I inadvertently duplicated this here: https://github.com/grafana/grafana/discussions/40486

I go back and forth between bug and feature request. It seems like it should work (bug) but it clearly doesn't and is actually a different use case (feature).

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
